lib/galaxy/dependencies/conditional-requirements.txt

```diff
@@ -8,7 +8,8 @@ drmaa
 statsd
 docker
 azure-storage==0.32.0
-python-irodsclient==0.8.3
+# python-irodsclient==0.8.3
+python-irodsclient@https://github.com/kxk302/python-irodsclient/archive/master.zip
```

Comment:
mhh, the '@https://' part gets ignored, so that doesn't pick up the right dependency

Reply:
Added whats in dev-requirements.txt here, in place of the commented out line.

Reply:
BTW, I changed the protocol to http